cmd/gardenlet/app/app.go

```diff
@@ -451,6 +465,133 @@ func (g *garden) Start(ctx context.Context) error {
 	return nil
 }
 
+// TODO(aaronfern): Remove this code after v1.91 has been released.
+func (g *garden) createNewDWDResources(ctx context.Context, seedClient client.Client) error {
+	// Fetch all namespaces
+	namespaceList := &corev1.NamespaceList{}
```

Comment:
Add a label selector for shoot namespaces only (`gardener.cloud/role=shoot`). Use constants in `v1beta1constants` package.